### DISCUSSION TITLE
Ktor client or Ktorfit for new KM project?

### DISCUSSION THREAD

**Discussion by nathanmeade** [Foso/Ktorfit#213] [Q&A] [answered]
I'm new to Kotlin Multiplatform as an Android developer and I am used to using Retrofit. If I'm just trying to get simple networking calls made with my KM apps. Trying to understand the difference between this and just a normal Ktor client.

**Comment by Foso** [comment, marked as answer]
Hi, with Ktorfit you can define your network API based on interfaces and annotations similar to the way Retrofit works. And it can be used with KM Projects. Ktorfit is built on top of Ktor Clients, similar to how Retrofit uses OkHttp. When you want to keep using something similar to Retrofit but in KM Projects, use Ktorfit. When you want to avoid code generation, use just Ktor clients.

When you want to try Ktorfit, please take a look at the example projects https://github.com/Foso/Ktorfit/tree/master/example/

**Reply by nathanmeade** [reply]
Thank you for clearing this up for me! @Foso
